### ISSUE TITLE
Dispatch > Work Assignments > View Dashboard: Mouseover numbers not accessible

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12246]
On the View Dashboard page under Work Assignments, the elements Hourly, Daily, Weekly and Monthly have mouse over functions to review and reveal data per hour for 24 hours. None of this information can be accessed with a keyboard or read by assistive technology programs. This information can be found on all four tabs displayed visually. (New: 08/27/2019) when with the mouse in the boxes “all” “full grants” “partial grants & remands”  the numbers change as I move my mouse there is no way to get in there with the keyboard and get the information.

<img width="1089" alt="Screen Shot 2019-09-27 at 5 20 54 PM" src="https://user-images.githubusercontent.com/1198106/65802973-3f299f80-e14b-11e9-984d-85e4d0ca392d.png">

**Comment by carodew** [comment]
Closing this since this will also be resolved with #12991
